libs/components/lists/src/lib/modules/repeater/repeater.component.ts

```diff
@@ -401,16 +401,14 @@ export class SkyRepeaterComponent
     const hasInteraction =
       this.reorderable ||
       this.items?.some((item) => item.isCollapsible) ||
+      this.items?.some((item) => !!item.selectable) ||
       !!(this.#elementRef.nativeElement as HTMLElement).querySelector(
         interactionSelector
       );
 
     if (hasInteraction) {
       // If the repeater matches interaction selector https://www.w3.org/TR/wai-aria-practices-1.1/#grid
       autoRole = 'grid';
-    } else if (this.items?.some((item) => !!item.selectable)) {
-      // If the only interaction is select https://www.w3.org/TR/wai-aria-practices-1.1/#Listbox
-      autoRole = 'listbox';
```

Comment:
Interactive elements should always be wrapped in a `grid` role, which includes selectable checkbox repeaters.